internal/env/env.go

```diff
@@ -1,10 +1,61 @@
 package env
 
 import (
+	"errors"
+	"fmt"
 	"os"
+	"strconv"
 	"strings"
 )
 
+const (
+	// ------------------------
+	//          auth
+	// ------------------------
+
+	// EnvVarAuthToken is the env var for the auth token sent by the launcher to
+	// the main instance in exchange for a single-use grant token.
+	// nolint:gosec // G101: False positive
+	EnvVarAuthToken = "N8N_RUNNERS_AUTH_TOKEN"
+
+	// EnvVarGrantToken is the env var for the single-use grant token returned by
+	// the main instance to the launcher in exchange for the auth token.
+	// nolint:gosec // G101: False positive
+	EnvVarGrantToken = "N8N_RUNNERS_GRANT_TOKEN"
+
+	// ------------------------
+	//        n8n main
+	// ------------------------
+
+	// EnvVarMainServerURI is the env var for the URI of the n8n main instance's
+	// main server.
+	EnvVarMainServerURI = "N8N_MAIN_URI"
+
+	// EnvVarMainRunnerServerURI is the env var for the URI of the n8n main
+	// instance's runner server. Do not confuse with `N8N_RUNNER_URI`.
+	EnvVarMainRunnerServerURI = "N8N_RUNNERS_N8N_URI"
```

Comment:
This is pretty confusing. Maybe we should start calling this the "Task Broker' to distinguish it from the other components. Then this would be `EnVarTaskBrokerURI` and `N8N_TASK_BROKER_URI`.

In hindsight it would have probably made more sense to not mix the n8n uri here and instead add the healthcheck endpoint for the task broker server. That way the broker would have been kept more independent. We can revisit this in the future if we deem so.